schema/home/home_page_module.js

```diff
@@ -4,15 +4,32 @@ import Results from './results';
 import Title from './title';
 import Context from './context';
 import Params from './params';
+import { NodeInterface } from '../object_identification';
+import { toGlobalId } from 'graphql-relay';
 import {
   GraphQLObjectType,
   GraphQLString,
   GraphQLBoolean,
+  GraphQLID,
+  GraphQLNonNull,
 } from 'graphql';
 
-export const HomePageModulesType = new GraphQLObjectType({
-  name: 'HomePageModules',
+export const HomePageModuleType = new GraphQLObjectType({
+  name: 'HomePageModule',
+  interfaces: [NodeInterface],
```

Comment:
In the future it’s possible that Relay would check if types _actually_ specify that they implement the `Node` interface, but at the moment it doesn’t really do that.